merlin/models/tf/inputs/embedding.py

```diff
@@ -62,7 +62,7 @@
 
 
 class EmbeddingTableBase(Block):
-    def __init__(self, dim: int, col_schema: ColumnSchema, trainable=True, **kwargs):
+    def __init__(self, dim: int, col_schema: ColumnSchema, *, trainable=True, **kwargs):
```

Comment:
I have just a question here.
Does the `*,` allows providing a list to `col_schema`?
Is it equivalent to passing `*col_schema`?
We could have a test demonstrating how multiple col schemas can be passed to `EmbeddingTable` (in addition to the `embedding_table.add_feature()` option)

Reply:
the `*` here forces the subsequent parameters to be passed as keyword-only (https://peps.python.org/pep-3102/). Without this, it would be valid to pass `trainable` as a positional parameter (without the name). 

The * is an 'unnamed' varargs. So you can't use it along side `*col_schema` here. It's one or the other. Both will turn subsequent params into keyword-only params.

_Related note_
There's also a way to make some args positional-only with `/`. That may be useful for some places in our codebase too.However, it was introduced in Python 3.8 (https://peps.python.org/pep-0570/), so would force 3.8 to be the minimum python we support. (We're only running tests on 3.9 currently, so there may already be some things we're using in Python that implicitly have pushed our min-version to at least 3.8)

Reply:
I've updated to support passing multiple column schemas to the __init__. And implemented support for the `schema` property.